README.md

```diff
@@ -89,6 +89,20 @@ VUE_APP_FIREBASE_MEASUREMENT_ID="xxxx"
 
 You will get the values if you create a web application for Google Firestore.
 
+### Configuration of Covid-Statistics
+
+You have to add `.env` file in the root directory having following environment variable
+to show statistics of your (German) Landkreis:
+
+```javascript
+VUE_APP_COVID_STATISTIC_OBJECT_ID_LIST="211,119,27,98,200"
```

Comment:
> Macht es Sinn, das als Umgebungsvariable mitzugeben? Sollte doch eher direkt hinterlegt werden.

To be configurable for other users/companies it could make sense (Open Source).

Reply:
> Sollten die Gebiete 210 (Tuttlingen), 209 (Schwarzwald-Baar), 220 (Bodenseekreis) und 213 (Waldshut) nicht auch angezeigt werden? Es kommen ja schließlich nicht alle aus dem Landkreis Konstanz.

May be we extend in further releases to allow user dependent areas.